rusk/src/lib/services/state.rs

```diff
@@ -113,68 +84,45 @@ impl Rusk {
 
     fn accept_transactions(
         &self,
-        request: Request<StateTransitionRequest>,
+        block_height: u64,
```

Comment:
This being a service implementation, wouldn't make more sense to accept a request instead of arbitrary arguments?